.github/workflows/update_docker_versions.yaml

```
@@ -4,7 +4,7 @@ name: Update Pulumi Docker image versions
 # Run workflow on pushes to matching branches
 on:  # yamllint disable-line rule:truthy
   push:
-    branches: [python-migration]
+    branches: [develop]
   schedule:
     - cron: "0 0 */7 * *"  # run once per week
```

```suggestion
    - cron: "0 3 * * 1"  # run at 3:00 every Monday
```

I think what is currently there will run when `day%7==0` instead of every seven days.

---

Even worse - I think it's "At 00:00 on every 7th day-of-month."!